cmd/kubeadm/app/util/config/testdata/defaulting/master/defaulted_v1alpha1.yaml

```diff
@@ -1,78 +0,0 @@
-api:
```

Comment:
this change seems unrelated?

Reply:
It's https://github.com/kubernetes/kubernetes/pull/64068/commits/4f0020d1b4caec992460859ceb792f1b785a85fd, I had to remove it as I realized kubeadm v1.11 won't be able to marshal the v1alpha1 API, only the newer v1alpha2 (this due to that conversions only work one way: from v1alpha1 to v1alpha2(==internal), not the other way round).
Hence I had to remove the e2e unit tests that printed v1alpha1 versions as they won't work.